flow-typed/npm/jest_v20.x.x.js

```diff
@@ -214,7 +214,7 @@ type JestExpectType = {
   /**
    * Use .toMatch to check that a string matches a regular expression.
    */
-  toMatch(regexp: RegExp): void,
+  toMatch(regexp: RegExp | string): void,
```

Comment:
The latest def for jest has this -- but also has more updates that break our code elsewhere, so I don't want to update it here.